src/app/pages/confirm-test-details/confirm-test-details.page.ts

```diff
@@ -208,7 +208,6 @@ export class ConfirmTestDetailsPage extends PracticeableBasePageComponent {
   }
 
   displayForCategory = (category: TestCategory): boolean => isAnyOf(category, [
-    TestCategory.ADI2,
```

Comment:
This method controls multiple fields, i.e. provisional licence, transmission and d255. Presumably this change will now mean the other fields will show for ADI2